presto-main/src/main/java/com/facebook/presto/type/UuidType.java

```diff
@@ -159,4 +160,21 @@ public final Slice getSlice(Block block, int position)
                 block.getLong(position, 0),
                 block.getLong(position, SIZE_OF_LONG));
     }
+
+    public static Slice javaUuidToPrestoUuid(UUID uuid)
+    {
+        return Slices.wrappedLongArray(
```

Comment:
`import static io.airlift.slice.Slices.wrappedLongArray;`

And then directly use `wrappedLongArray(...`